spec/models/miq_expression_spec.rb

```diff
@@ -1631,4 +1631,100 @@
       end
     end
   end
+
+  describe ".get_col_operators" do
+    subject { described_class.get_col_operators(@field) }
+
+    it "returns array of available operations if parameter is :count" do
+      @field = :count
+      expect(subject).to eq(["=", "!=", "<", "<=", ">=", ">"])
+    end
+
+    it "returns list of available operations if parameter is :regkey" do
+      @field = :regkey
+      expect(subject).to eq(["=",
```

Comment:
Is the order of elements important here? If not you may want to use `contain_exactly`